### PR TITLE
Various flaking/code cleanup

### PR DESCRIPTION
**Pull request by miigotu**
Change log from debug to info for http status code returned from clients
Change log from error to warning when sending to client failed (the request failed, client down or network error maybe)

### CODE REVIEW

**Comment by SickRage2** [comment]
Thanks for the pull request! Before a real human comes by, please make sure your PR has all of the below criteria checked
- [x] Give a description on what the PR is for.
- [x] Make sure your PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review

Please make sure you also read [contribution guide](https://github.com/SickRage/SickRage/blob/master/contributing.md) and followed all of the steps.

Thanks!

File: sickbeard/failed_history.py [hunk 1]
```diff
@@ -223,14 +223,15 @@ def findRelease(epObj):
 
     # Clear old snatches for this release if any exist
     failed_db_con = db.DBConnection('failed.db')
```
Comment (fernandog) [inline sickbeard/failed_history.py:225]: @miigotu do you know why we clean old snatches? Can't understand the reason. deleting it maybe the cause of N/D quality in history when make as failed

Comment (miigotu) [inline sickbeard/failed_history.py:225]: I was asking myself the same thing. i think we can comment that part and see if it fixes it in develop. Will do it now, maybe you can test it also?

Comment (fernandog) [inline sickbeard/failed_history.py:225]: np!

**Pull request merged**